spark/v3.4/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestBranchDDL.java

```diff
@@ -289,4 +300,8 @@ private Table insertRows() throws NoSuchTableException {
     df.writeTo(tableName).append();
     return validationCatalog.loadTable(tableIdent);
   }
+
+  private Table createEmptyTable() {
+    return validationCatalog.loadTable(tableIdent);
```

Comment:
this is not create table but rather load a table from catalog. 

I think empty table creation is already handled in `before` method at line 43 so this method is not needed at all